OutOfSchool/OutOfSchool.BusinessLogic/Models/Workshops/StudyPeriodDatesDto.cs

```diff
@@ -3,14 +3,28 @@
 
 namespace OutOfSchool.BusinessLogic.Models.Workshops;
 
-public class StudyPeriodDatesDto
+public class StudyPeriodDatesDto : IValidatableObject
 {
     [Required(ErrorMessage = "Study period start date is required")]
     public DateOnly StartDate { get; set; }
 
 
     [Required(ErrorMessage = "Study period end date is required")]
     public DateOnly EndDate { get; set; }
+
+    public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
```

Comment:
need to double check this, as we only care for `day` & `month`. The year can be any, which may lead to situations where:
- start: `01-09-2025`, end: `31-05-2026`
- start: `01-09-2025`, end: `31-05-2024`

are both valid technically, as the _year_ part is up to user imagination due to understanding of how academic groups work. So we do not show it to end users